presto-product-tests/conf/presto/etc/multinode-tls-ldap-master.properties

```diff
@@ -0,0 +1,30 @@
+#
+# WARNING
+# ^^^^^^^
+# This configuration file is for development only and should NOT be used be
+# used in production. For example configuration, see the Presto documentation.
+#
+
+node.id=will-be-overwritten
+node.environment=test
+node.internal-address-source=FQDN
+
+coordinator=true
+node-scheduler.include-coordinator=true
```

Comment:
once you set it to `false` the tests are gonna start to fail due to the thing i explained above. But when the scheduling to a coordinator is enabled, despite it cannot communicate with workers the test will still pass, as there is a one node to run the queries